### ISSUE TITLE
Old forum link references to update

### ISSUE THREAD

**Issue by Arthmoor** [wrye-bash/wrye-bash#435]
The following files have references to the old forums at bethsoft.com which have long since been shut down:

Mopy\bash\l10n\Chinese (Simplified).txt
Mopy\bash\l10n\Chinese (Traditional).txt
Mopy\bash\l10n\Russian.txt
Mopy\bash\patcher\patchers\races_multitweaks.py
Mopy\Docs\Wrye Bash Version History.html
Mopy\Docs\Wrye Bash Advanced Readme.html
Mopy\Docs\Wrye Bash General Readme.html

They all need to be updated to point to the new forum thread on AFK Mods: https://www.afkmods.com/index.php?/topic/4966-wrye-bash-all-games/

**Comment by Infernio** [comment]
Most of these were either left in on purpose (because they point to specific posts in the bethsoft thread) or got missed because of PyCharm's ignore settings (the translations).

> Mopy\bash\l10n\Chinese (Simplified).txt
> Mopy\bash\l10n\Chinese (Traditional).txt
> Mopy\bash\l10n\Russian.txt 

These got missed because PyCharm ignores the l10n folder. Not really a big problem, since the english lines from which the translations stem have changed. That means they'll just show as english, but with the correct link. Will fix.

> Mopy\bash\patcher\patchers\races_multitweaks.py

I can't find any occurrences of `bethsoft.com` in there. Are you sure you have the latest dev version? There _were_ two occurrences in there before the `inf-afkmods-update` merge, but I changed them to afkmods links in those commits.

> Mopy\Docs\Wrye Bash Version History.html

Link: http://forums.bethsoft.com/index.php?app=forums&module=forums&section=findpost&pid=17975809
(Ancient link format, I've updated it to http://forums.bethsoft.com/topic/1204328-relz-wrye-bash/page-3#entry17975809 now)
A post containing a request from speedyg869, which was implemented in that version.

> Mopy\Docs\Wrye Bash Advanced Readme.html

Link: http://forums.bethsoft.com/topic/1536340-wrye-bash-thread-105/page-3#entry24345148
A post providing some previous discussion on the 'Add Master' functionality. I still advocate that we should remove it _entirely_, because of how dangerous it is. If we did that, we could also drop the readme entry. If anyone wants to fix and resurrect it in the future, it'll still be in the git history.

>  Mopy\Docs\Wrye Bash General Readme.html

Link: http://forums.bethsoft.com/topic/1369136-thrashing-thread-1/?hl=+508
A thread containing information on directory thrashing in Oblivion.

I can't really change any of these last three - I guess I could replace them with archive.org mirrors of bethsoft.com?
